pyproject.toml

```diff
@@ -35,6 +35,7 @@ known_third_party = ["click","google","kedro","kfp","kubernetes","tabulate", "py
 python = ">=3.8,<3.11"
 kedro = ">=0.18.1, <0.19.0"
 click = ">=8.0.4"
+docker = { version = "~=5.0.3", optional = true }
```

Comment:
I'm not sure if we need docker SDK if `subprocess` is in use

Reply:
Yes, forgot about it, thanks